tests/Functional/Controller/Admin/SecurityControllerTest.php

```diff
@@ -72,8 +77,12 @@ public function testNewAction(): void
         $crawler = $this->client->request('GET', '/admin/security/new');
         self::assertEquals(200, $this->client->getResponse()->getStatusCode());
         $form = $crawler->selectButton('Toevoegen')->form();
-        $form['local_account[name]'] = 'John';
-        $form['local_account[email]'] = 'john@doe.eyes';
+        $form->setValues([
+            'local_account[name]' => 'John',
+            'local_account[email]' => 'john@doe.eyes',
+        ]);
+        //$form['local_account[name]'] = 'John';
+        //$form['local_account[email]'] = 'john@doe.eyes';
```

Comment:
Please clean up commented out code, as it's stored in the git history.

Reply:
my bad, think I was checking the new syntax and forgot